src/backend/parser/analyze.c

```diff
@@ -1926,6 +1946,20 @@ transformSetOperationStmt(ParseState *pstate, SelectStmt *stmt)
 										  EXPR_KIND_ORDER_BY,
 										  false /* allow SQL92 rules */ );
 
+	/* 
```

Comment:
We should add a hook type named : post_transfrom_sort_clause_hook
and move the code into babel